diesel/src/expression/sql_literal.rs

```diff
@@ -94,5 +181,81 @@ impl<ST> NonAggregate for SqlLiteral<ST> {}
 /// # }
 /// ```
 pub fn sql<ST>(sql: &str) -> SqlLiteral<ST> {
-    SqlLiteral::new(sql.into())
+    SqlLiteral::new(sql.into(), ())
+}
+
+#[derive(Debug, Clone, Copy)]
+#[must_use = "Queries are only executed when calling `load`, `get_result`, or similar."]
+pub struct UncheckedBind<Query, Value> {
+    query: Query,
+    value: Value,
+}
+
+impl<Query, Value> UncheckedBind<Query, Value>
+where
+    Query: Expression,
+{
+    pub fn new(query: Query, value: Value) -> Self {
+        UncheckedBind { query, value }
+    }
+
+    pub fn sql(self, sql: &str) -> SqlLiteral<Query::SqlType, Self> {
+        SqlLiteral::new(sql.into(), self)
+    }
+}
+
+impl<Query, Value> Expression for UncheckedBind<Query, Value>
+where
+    Query: Expression,
+{
+    type SqlType = Query::SqlType;
+}
+
+impl<Query, Value> QueryId for UncheckedBind<Query, Value>
```

Comment:
This impl is wrong. We should just derive it.

Reply:
Any examples of this elsewhere?

Reply:
Just put a `#[derive(QueryId)]` on the definition of `UncheckedBind`

Reply:
Ack, I totally forgot about this!